tools/releaseBuild/azureDevOps/templates/upload.yml

```diff
@@ -34,3 +35,19 @@ steps:
     storage: '$(StorageAccount)'
     ContainerName: '$(AzureVersion)'
   condition: and(succeeded(), eq(variables['Build.Reason'], 'Manual'))
+
+- template: upload-final-results.yml
+  parameters:
+    artifactPath: $(Build.StagingDirectory)\signedPackages
+    artifactFilter: PowerShell-${{ parameters.version }}-win-${{ parameters.architecture }}.msix
```

Comment:
you are going to have to calculate your own version because the MSIX version is not the same as other packages versions

Reply:
What will this look like when we are preview?

Reply:
Yes, will have to update this.

Reply:
I'm just going to have the package name match what we call the msi package.  It's only the package version that will differ due to requirements for appx.